server/routes.go

```diff
@@ -72,6 +74,11 @@ func GenerateHandler(c *gin.Context) {
 			loaded.digest = ""
 		}
 
+		if model.Embeddings != nil && len(model.Embeddings) > 0 {
+			opts.EmbeddingOnly = true // this is requried to generate embeddings, completions will still work
```

Comment:
Is this flag helpful to have, then?

Reply:
(side note, not critical for this PR)

Reply:
changed this to be enabled by default, doesnt look like it will have an effect on speed to have it on